cmd/core/main.go

```diff
@@ -172,6 +175,10 @@ func main() {
 		setupLog.Error(err, "unable to setup the vela core controller")
 		os.Exit(1)
 	}
+	if err = storage.NewStorage(storageDriver); err != nil {
```

Comment:
How does local drive work in a controller? On the flip side, where is app/appFile used on the server side?

Reply:
1. We can start the environment variable configuration in two ways(Default: Local):
   a) first use system environment
   b) startup arguments from storage-driver
2.  If local storage is used, it will still be stored under ` .vala`. It should also support customization and split with CLI.
3. It is preferred to use [it](https://github.com/oam-dev/kubevela/blob/master/pkg/application/app.go) to complete the application action, and it is not recommended to use `appfile/storage` pkg.